src/server/network/protocol/protocolgame.cpp

```diff
@@ -1369,9 +1364,12 @@ void ProtocolGame::parsePacketFromDispatcher(NetworkMessage msg, uint8_t recvbyt
 			// case 0xDF, 0xE0, 0xE1, 0xFB, 0xFC, 0xFD, 0xFE Premium Shop.
 
 		default:
-			std::string hexString = fmt::format("0x{:02x}", recvbyte);
-			g_logger().debug("Player '{}' sent unknown packet header: hex[{}], decimal[{}]", player->getName(), asUpperCaseString(hexString), recvbyte);
-			break;
+			// Modules system
+			if (!g_modules().executeOnRecvbyte(player->getID(), msg, recvbyte)) {
```

Comment:
This check: ```recvbyte != 0xD3``` is still necessary, because this byte is checked in another place (in the outfit window)
https://github.com/opentibiabr/canary/blob/3387b561c035c2708b6d4c7b1b5a5ae3618a2a7b/src/server/network/protocol/protocolgame.cpp#L1664

Reply:
In "`recvbyte != 0xD3`" the byte is not sent to `executeOnRecvbyte` and within the case it will also not reach `executeOnRecvbyt`, only within `parseSetOutfit`, as it already is, so the change does not change the functioning of this part.